content/messages/messages-fr.yaml

```diff
@@ -0,0 +1,120 @@
+or: "ou"
+projects: "Projets"
+how_permissions: "Comment paramétrer les permissions ?"
+help_center_in_your_language: "Le help center est maintenant en français !"
+help_center_in_your_language_description: "Mais pas trop quand même... Et oui ! Il reste des articles à traduire. Comme toujours, nous, on croit à la puissance de l'open source. Du coup, si vous êtes arrivés au bout de vos séries Netflix et que vous avez du temps à tuer, n'hésitez pas à nous donner un coup d'main !"
+its_here: "On vous explique tout comment faire ici."
+using_akeneo: "Utiliser<br>Akeneo"
+using_akeneo_description: "Découvrez les concepts clés. Maîtrisez vos données produit. Gagnez en productivité. Travaillez en équipe."
+browse_all_articles: "Accéder aux articles"
+setting_up_the_pim: "Paramétrer<br>le PIM"
+setting_up_the_pim_description: "Définissez votre structure de catalogue. Utilisez la puissance des permissions. Suivez l'évolution de vos jobs."
+coding_around: "Coder autour<br>d'Akeneo"
+coding_around_description: "Tirez profit de la magie de l'API. Domptez la customisation de votre PIM. Suivez notre guide de style."
+dev_hub_link: "En route pour le Dév hub"
+popular_guides: "Guides populaires"
+need_more_help: "Un projet PIM assez complexe ?"
+need_more_help_descrition: "Que vous ayez des soucis pour démarrer dans le PIM ou pour modéliser votre structure de catalogue, l'Akademy est là pour vous aider à vous former. N'hésitez pas à nous contacter!"
+get_trained: "Une formation, une !"
+stay_in_touch_community: "Plongez au sein de notre communauté"
+slack_user_group: "Groupe d'utilisateurs Slack"
+slack_user_group_description: "Rejoignez notre communauté Slack"
+stack_overflow_description: "Une question technique ? C'est par ici !"
+crowdin_description: "Notre PIM est multilingue ! Aidez-nous à le traduire dans de nombreuses langues."
+badger_description: "De grands accomplissements = de magnifiques badges! Venez les réclamer ici."
+stay_in_touch_us: "Toujours proche de nous"
+blog: "Blog"
+blog_deccription: "Pour suivre notre actualité, abonnez-vous au blog"
+contact_us: "Contactez-nous !"
+contact_us_description: "Besoin de nous contacter ? Laissez-nous un message !"
+welcome_partners: "Nos très chers partenaires !"
+welcome_partners_description: "Votre portail dédié, c'est par là."
+more_curious: "Toujours aussi curieux ?"
+changelog_of_v2: "Changelog de la v2.0"
+webinars_of_v2: "Webinars de la v2.0"
+userguides_for_v16_17: "Guides utilisateur v1.6 & v1.7"
+marketplace: "Marketplace"
+customer_stories: "Cas client"
+knowledge_base: "Base de connaissance"
+coding: "Vous souhaitez coder. Bien ! Mais quoi, plus précisément ?"
+build_connectors: "Construire des<br> connecteurs !"
+api_friend: "L'API est votre ami"
+api_friend_description: "Connectez votre PIM avec des ERP, DAM, solutions print, plateformes e-commerce, le monde entier quoi ! Tout est possible !"
+go_API: "Doc de l'API"
+pimp_my_pim: "Pimper<br>mon PIM !"
```

Comment:
Tuner mon PIM ?

Reply:
I prefer "Pimper" for the play on words.